gulpfile.js

```diff
@@ -65,6 +67,9 @@ gulp.task('pug', () => {
 gulp.task('styl', () => {
 	return gulp.src('src/styl/index.styl')
 		.pipe(stylus({compress: true}))
+		.pipe(postcss([
+			autoprefixer({browsers: ['last 1 version']})
```

Comment:
I think we should do at least `last 2 versions`

Reply:
Done